### PR TITLE
Kubernetes tutorial: Added details in Install pod network add-on section

### PR DESCRIPTION
**Pull request by MCamp859**
Reviewers: @mythi has provided content, which includes fix for [clearlinux/clr-bundles#81 issue](https://github.com/clearlinux/clr-bundles/issues/81)

### CODE REVIEW

File: source/clear-linux/tutorials/kubernetes.rst [hunk 1]
```diff
@@ -203,13 +203,36 @@ communicate.
...
+
+..  code-block:: bash
+
+    plugin_dir = "/usr/libexec/cni/"
+
+to:
+
+..  code-block:: bash
+
+    plugin_dir = "/opt/cni/bin"
```
Comment (eadamsintel) [inline source/clear-linux/tutorials/kubernetes.rst]: I also had to symbolically link my loopback directory for weave to work. There might be a different more appropriate way to do this though but this worked for me.

ln –s /usr/libexec/cni/loopback /opt/cni/bin/loopback

Comment (MCamp859) [reply source/clear-linux/tutorials/kubernetes.rst]: @eadamsintel  Please look at the last commit, lines 248-252 contain the ln command.

Comment (eadamsintel) [reply source/clear-linux/tutorials/kubernetes.rst]: Ok I missed that.  Still, I don't know that this is the best way to do this but it works for now. If Clear Linux gets updated where this step is no longer needed then we can remove at that point.

**Comment by mvincerx** [comment]
Thanks for the dedicated collaboration, all. Great improvements for the network plug-ins and user options.

**Pull request merged**